client/src/components/ExploreCard.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
```

Comment:
This card is from another feature I think no? In that case make sure you've merged dev into your branch so you have the latest versions of these files. Typically that's best practice before making a PR so that the PR stays up to date, and then keep merging the latest dev into this branch as dev gets updated as well.